lib/generators/active_admin/resource/templates/admin.rb.erb

```diff
@@ -1,11 +1,12 @@
 ActiveAdmin.register <%= class_name %> do
-# See permitted parameters documentation:
-# https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+# Uncomment all parameters which should be permitted for assignment
```

Comment:
Any reason to invert the order of the document reference and the boilerplate code here?

Reply:
Sorry I did not have time to respond. I felt the permit_params is the most relevant thing here in the template. Did any of this change make it into master? Should I start over with this (that's fine).

Reply:
No, I should apologize!

I was editing your PR myself because I changed the template's indentation so it got conflicts. And when doing that I unintentionally force-pushed activeadmin's master branch to your fork, so the PR got closed automatically because it no longer contains any changes. I'll open a separate PR, sorry about that.